hive/build.gradle

```diff
@@ -58,6 +58,12 @@ dependencies {
     javadocSources(project(":elasticsearch-hadoop-mr"))
 }
 
+tasks.named("test").configure {
+    onlyIf { JavaVersion.current() >= JavaVersion.VERSION_16 }
```

Comment:
Is this just needed to conditionally add the jvmArgs settings or are these tests required to run on JVM 16 and up?

Reply:
This is just for the jvmArgs settings. The tests still run for all versions right?

Reply:
I think the `onlyIf` method marks the task to only run when the condition is true